src/compass/utils/geometry_utils.py

```diff
@@ -265,3 +266,122 @@ def get_unit_vector4component_of_interest(los_inc_angle, los_az_angle, comp='enu
         ]
 
     return unit_vec
+
+def enu2rgaz(radargrid_ref, orbit, ellipsoid,
```

Comment:
The conversion in this PR is based on `geo2rdr`, meaning that it will calculate range and azimuth displacements simultaneously. I'm not sure how much useful separating them would be.
In case we want to use only one of the components, I think we can suppress the unnecessary return values. For example,

```python
rg_arr, _ = enu2rgaz(radargrid_ref, orbit, ellipsoid, ...)
_, az_arr = enu2rgaz(radargrid_ref, orbit, ellipsoid, ...)

```